recipes/newrelic/infrastructure/debian.yml

```diff
@@ -145,15 +145,15 @@ install:
       # apt will return an error if fails to update any of its sources. Ignore these errors and let the "install_infra" task fail.
       ignore_error: true
 
-    add_gnupg2_if_required:
+    install_gnupg:
       cmds:
         - |
-          if [ $(({{.DEBIAN_VERSION}})) -ge 10 ]; then
+          if [ {{.HAS_GPG}} -eq 0 ] ; then
```

Comment:
Should we also keep the Debian version check? (`if [ $(({{.DEBIAN_VERSION}})) -ge 10 ]`) I'm a little unfamiliar with this specific task in the recipe, but at least wanted to ask 🙂

Reply:
if gpg is not installed, it's going to fail whatever the version. I tried that on debian jessie, and it worked. To be honest, I'm not sure why we have this version check in the first place. Perhaps @Julien4218 knows

Reply:
super